resources/views/permissions.blade.php

```diff
@@ -68,29 +68,30 @@ public function __construct()
      */
     public function up(): void
     {
-        DB::transaction(function () {
+        $tableNames = config('permission.table_names');
```

Comment:
I would prefer to have some fallback to original names if config is not present. Just to be sure it will work.